.ruff.toml

```diff
@@ -17,6 +16,7 @@ exclude = ["setup/*"]
 [per-file-ignores]
 "__init__.py" = ["F401", "I001"]  # ignore unused and unsorted imports in __init__.py
 "__manifest__.py" = ["B018"]  # useless expression
+"base_report_to_printer/models/printing_server.py" = ["B023"]
```

Comment:
You should add the exception in the module file itself, not in the repo template

Reply:
Thanks for the hint. I will change it later today. I am not used to ruff.

Reply:
@chienandalu can you give me a hint how to do this?
I added .ruff.toml inside base_report_to_printer and added 
```
[per-file-ignores]
"models/printing_server.py" = ["B023"]
```
But tests fail now. How to do it in modules?

Reply:
Well, you don't need to change the `ruff.toml` file at all.

To add an exception to a false positive you go to the affected line and put in a comment at the end of it or above it:
```py
# noqa: B023
```

this might work as well

```py
# ruff: noqa: B023
```

As for the tests, it must be something else, as lint errors shouldn't have any effect on it

Reply:
Thanks, it works.